brats/training/runners.py

```diff
@@ -33,8 +33,9 @@ def run_training_epoch(model: nn.Module, data_loader: data.DataLoader, optimizer
         optimizer.step()
         losses.append(loss.item())
         for metric_name in metrics:
-            out_metrics[metric_name].append(metrics[metric_name](output, target).item())
-    out_metrics = {metric_name: np.mean(out_metrics[metric_name]) for metric_name in metrics.keys()}
+            metric_values = [value.item() for value in metrics[metric_name](output, target)]
```

Comment:
This is great, but if I understand correcltly, it will fail if a metric returns a scalar value.  The question is: do we care?

Reply:
changed how metrics are calculated